app/views/drugs/_drug_summary_tab.html.haml

```diff
@@ -0,0 +1,54 @@
+/ takes in variable for type of summary (drug or gene)
+/ displays lists of aliases, drug/gene info, publications
+/ figure out how to get "card" aesthetic
+
+%div(class="item-content")
+	%div{class: "cell category", style: "float: right; width: 50%"}
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.